deeppavlov/configs/classifiers/dnnc.json

```
@@ -0,0 +1,131 @@
+                    "x_support",
+                    "x_populated",
+                    "y_support"
+                ],
+                "x": "text",
+                "y": "category",
+                "format": "json",
+                "orient": "split",
+                "support_dataset_path": "{SUPPORT_DATA_PATH}/parsed_dataset.json"
+            },
+            {
+                "class_name": "torch_transformers_preprocessor",
+                "in": [
+                    "x_populated",
+                    "x_support"
+                ],
+                "out": ["bert_features"],
+                "vocab_file": "{BASE_MODEL}",
+                "do_lower_case": true,
+                "max_seq_length": 128
+            },
+            {
+                "class_name": "torch_transformers_classifier",
+                "main": true,
+                "in": ["bert_features"],
+                "out": ["simmilarity_scores"],
+                "n_classes": 2,
+                "return_probas": true,
+                "pretrained_bert": "{BASE_MODEL}",
+                "save_path": "{MODEL_PATH}/model",
+                "load_path": "{MODEL_PATH}/model",
+                "is_binary": "{BINARY_CLASSIFICATION}",
+                "optimizer_parameters": {
+                    "lr": 2e-05
+                }
+            },
+            {
+                "class_name": "dnnc_proba2labels",
+                "is_binary": "{BINARY_CLASSIFICATION}",
+                "in": [
+                    "simmilarity_scores",
+                    "x",
+                    "x_populated",
+                    "y_support"
+                ],
+                "out": ["y_pred"],
+                "multilabel": false,
+                "confidence_threshold": 0.0
+            }
+        ],
+        "out": ["y_pred"]
+    },
+    "train": {
+        "batch_size": 10,
+        "metrics": [
+            {
+                "name": "accuracy_oos",
+                "inputs": [
+                    "y_true",
+                    "y_pred"
+                ],
+                "exclude_oos": true
+            },
+            {
+                "name": "balanced_accuracy_oos",
+                "inputs": [
+                    "y_true",
+                    "y_pred"
+                ],
+                "exclude_oos": true
+            },
+            {
+                "name": "oos_scores",
+                "inputs": [
+                    "y_true",
+                    "y_pred"
+                ]
+            }
+        ],
+        "show_examples": false,
+        "evaluation_targets": ["test"],
+        "class_name": "torch_trainer"
+    },
+    "metadata": {
+        "variables": {
+            "ROOT_PATH": "~/.deeppavlov",
+            "DOWNLOADS_PATH": "{ROOT_PATH}/downloads",
+            "SUPPORT_DATA_PATH": "{ROOT_PATH}/parsed_datasets",
+            "MODELS_PATH": "{ROOT_PATH}/models",
+            "MODEL_PATH": "{MODELS_PATH}/classification/roberta_nli_binary_finetune",
```

```suggestion
            "MODEL_PATH": "{ROOT_PATH}/models/classification/roberta_nli_binary_finetune",
```
since MODELS_PATH used only in MODEL_PATH